R/controls.R

```diff
@@ -79,33 +127,31 @@ get_model_controls <- function() {
   )
 
   list(
-    area_scope = list(
+    area_scope = control(
       name = "area_scope",
       type = "multiselect",
       required = TRUE
     ),
-    area_level = list(
+    area_level = control(
       name = "area_level",
       type = "select",
       required = TRUE
     ),
-    calendar_quarter_t1 = list(
+    calendar_quarter_t1 = control(
       name = "calendar_quarter_t1",
       type = "select",
       required = TRUE
     ),
-    calendar_quarter_t2 = list(
+    calendar_quarter_t2 = control(
       name = "calendar_quarter_t2",
       type = "select",
-      required = TRUE,
-      value = "CY2021Q4"
+      required = TRUE
     ),
-    calendar_quarter_t3 = list(
+    calendar_quarter_t3 = control(
       name = "calendar_quarter_t3",
       type = "select",
       required = TRUE,
-      value = "CY2022Q3",
-      helpText = t_("OPTIONS_OUTPUT_PROJECTION_QUARTER_HELP"),
+      help_text = t_("OPTIONS_OUTPUT_PROJECTION_QUARTER_HELP"),
```

Comment:
Thanks, done that now for help text and label. Not done it for the `option` labels because some of these are not just straight translation keys e.g. `paste(t_("MONTH_MARCH"), "2020")`

Reply:
sounds good!